guide/using-the-pioneer-3-at-robot.md

```diff
@@ -0,0 +1,47 @@
+## Using the Pioneer 3-AT robot
+
+%figure "Pioneer 3-AT, a ready-to-use all terrain base"
+
+![pioneer3at_real.png](images/pioneer3at.png)
+
+%end
+
+The Pioneer 3-AT robot is an all-purpose outdoor base, used for research and prototyping applications involving mapping, navigation, monitoring, reconnaissance and other behaviors.
+It is caracterized by a set of features listed in [this table](#pioneer-3-at-characteristics).
+This model includes support for 4 motors and 16 sonar sensors (8 forward-facing, 8 rear-facing) for proximity measurements.
+
+More information on the specifications and optional devices is available on Adept Mobile Robots official [webpage](http://www.mobilerobots.com/ResearchRobots/ResearchRobots/P3AT.aspx).
+
+### Pioneer 3-AT model
+
+%figure "Pioneer 3-AT characteristics"
+
+| Characteristics     | Description                                                                     |
+| ------------------- | ------------------------------------------------------------------------------- |
+| Dimensions          | 508 mm long, 497 mm large, 277 mm high                                          |
+| Weight              | 12 kg, operating playload of 12 kg on floor                                     |
+| Speed               | Max. forward/backward speed: 2.2 m/s; Rotation speed: 20 rad/s                  |
+
+%end
+
+The standard model of the Pioneer 3-AT is provided in the "pioneer3AT.wbt" file which is located in the "WEBOTS\_HOME/projects/robots/pioneer/pioneer3at/worlds" directory of the Webots distribution.
```

Comment:
The up-to-date path is ".../projects/robots/adept/pioneer3/worlds/..." (to be fixed at several places)